src/workerd/io/worker-interface.capnp

```diff
@@ -147,6 +147,16 @@ struct QueueMessage @0x944adb18c0352295 {
   contentType @3 :Text;
 }
 
+struct QueueRetryBatch {
+  value @0 :Bool;
```

Comment:
What does `value` mean here? It's very unclear from just looking at this file what this is for, and ideally capnp schemas are the best place to start reading when trying to understand part of the system.

Could you rename this and/or add a comment?

Reply:
`value` indicates whether the batch should be retried. I will make it clear.